server/dataone_upload.py

```diff
@@ -230,7 +231,7 @@ def create_upload_package(item_ids, tale, user, repository):
 
     try:
         client = create_client(repository, {"headers": {
-            "Authorization": "Bearer eyJhbGciOiJSUzI1NiJ9.eyJzdWIiOiJodHRwOlwvXC9vcmNpZC5vcmdcLzAwMDAtMDAwMi0xNzU2LTIxMjgiLCJmdWxsTmFtZSI6IlRob21hcyBUaGVsZW4iLCJpc3N1ZWRBdCI6IjIwMTgtMDUtMTlUMTc6MTM6MTguMzgzKzAwOjAwIiwiY29uc3VtZXJLZXkiOiJ0aGVjb25zdW1lcmtleSIsImV4cCI6MTUyNjgxNDc5OCwidXNlcklkIjoiaHR0cDpcL1wvb3JjaWQub3JnXC8wMDAwLTAwMDItMTc1Ni0yMTI4IiwidHRsIjo2NDgwMCwiaWF0IjoxNTI2NzQ5OTk4fQ.g5lWjRqG71V0jaeOR8_v3KRWwkITQuVLt76_DYt6RvR9xVkLLdOcsB654SmbFZykrF7Hr4OityDXsu1U-eTRUvTTwsSlG3KyO1llmudcowFQ_CiYsTF0RJxIywwLhPbcXBWy8tjTgxA0Ni7rVNP4v_N8y9UQ1C0BvNVFp9RktLYik0u2tGYrdiotLIsVlNhvUwwjvrdHJ3QHlQIoi34TttIyss4O04iwVN9_wasS3YDrBQRWKkH3hJRcWQXHjcmd1-LUo7ek-cIL_ckcnmg4yV7Ods6FOrfK1Pz56IS-MWYAs1sJq5vWB2OxW1YGNryUwE5D7tf_bLdzO65BL-ZYfEvMUSnggliN6AxpX2dfGGEtBXAvcggX70jm0gR11zK46-bSQLf8ZLin0ZhtZTiPZ8X86NpJVZqmUUpEGU0dQGr5eSq9CrobdVbvyEPd4srwdiqfwVrafjJc-JevE0314VFhQjeko26UezPofUcXUmVT9x6Ydp-RiNdiAPImj3tnZTmPdZ-F8QFSoXj35PjCZbRUFk4SbFsl5OMHNfOo4pVNqSxJR14BW5OEA8UOVzYpdjes0-LsazkQmSsuCYj1F7Bn3lylxE9UqK4Jy1aNRFBs_DMrli2SCiqGf1iyfcme1R1hdSu0T8YZTkz4X6tcsN2fgepsJk6_rwGxg3kwu_o"}})
```

Comment:
Just FYI: your auth token is still in git. It's probably a test env  token and it looks like it expired May 20th so it's probably fine to leave in git. Thought I'd give you a heads up anyway.